README.md

```diff
@@ -55,7 +55,7 @@ This project is currently general availability.
 
 ## Internal Architecture
 
-<img src="./docs/assets/images/apisix-ingress-controller-arch.png" alt="module" width="600" height="313" />
+<img src="./docs/assets/images/apisix-ingress-controller-arch.png" alt="module" width="74.3%" height="55.9%" />
```

Comment:
May I know why you choose such a ratio?

Reply:
Refered this [file](https://github.com/apache/apisix-ingress-controller/pull/301/files#diff-b335630551682c19a781afebcf4d07bf978fb1f8ac04c6bf87428ed5106870f5R32), and I think a scaled image is more appropriate than a fixed height and width.

Reply:
Got it.